libbpf-tools/containers.c

```diff
@@ -17,3 +18,30 @@ struct container get_container_info(int map_fd, uint64_t mtnnsid) {
 
 	return container;
 }
+
+const char *NODE_HEADER = "NODE";
+const char *NAMESPACE_HEADER = "NAMESPACE";
+const char *POD_HEADER = "POD";
+const char *CONTAINER_HEADER = "CONTAINER";
+
+void print_container_info_header(void) {
+	/*
+	 * Format options used are the following:
+	 * '-' is used to left justify the data printed.
+	 * 16 is used to indicate the minimal width of the data printed, if data are
+	 * less than 16, spaces will be added.
+	 * '.16' is used to indicate the maximal number of characters to print, in
+	 * this case 16.
+	 *
+	 * These options ensures data printed will be 16 characters long left
+	 * justified.
+	 */
+	printf("%-16.16s %-16.16s %-16.16s %-16.16s", NODE_HEADER, NAMESPACE_HEADER,
+	       POD_HEADER, CONTAINER_HEADER);
+}
+
+void print_container_info(int map_fd, uint64_t mtnnsid) {
+	struct container c = get_container_info(map_fd, mtnnsid);
+	printf("%-16.16s %-16.16s %-16.16s %-16.16s", c.node, c.kubernetes_namespace,
```

Comment:
I think there should be a space after this line is printed, so the continuation line printed by the gadget is separated. 

How to test?
```
# Create a pod with a very long name
kubectl run mypodaaaaaaaaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbb --restart=Never -it --image=praqma/network-multitool -- /bin/sh

# run any of the bcc gadgets and execute a command inside the pod
./kubectl-gadget execsnoop 
NODE             NAMESPACE        POD              CONTAINER       PCOMM            PID    PPID   RET ARGS
ubuntu-hirsute   default          mypodaaaaaaaaaaa mypodaaaaaaaaaaacat              51690  51428    0 /bin/cat /dev/null
# 
```

Notice how the `CONTAINER` and `PCOMM` columns don't have a space in between. 

This problem **wasn't** introduced by this PR. Would you mind also checking if the python tools have this issue? 
Thanks!